xiblint/rules/strict_font_names.py

```diff
@@ -0,0 +1,26 @@
+from xiblint.rules import Rule
+
+
+class StrictFontNames(Rule):
+    """
+    Ensures fonts are in the allowed set.
+
+    Example configuration:
+    {
+      "allowed_fonts": ["ComicSans-Regular", "ComicSans-Bold"],
+      "allow_system_fonts": true
+    }
+    """
+    def check(self, context):  # type: (Rule, xiblint.xibcontext.XibContext) -> None
+        allowed_fonts = self.config.get('allowed_fonts', [])
+        allow_system_fonts = self.config.get('allow_system_fonts', True)
+
+        for element in context.tree.findall(".//fontDescription") + context.tree.findall(".//font"):
+            font_name = element.get("name")
+            if font_name is None:
+                if not allow_system_fonts:
+                    context.error(element, "Use of system fonts is not allowed.")
+                continue
+
+            if font_name not in allowed_fonts:
+                context.error(element, f'"{font_name}" is not one of the allowed fonts.')
```

Comment:
this syntax isn't valid for python 2.x, you can use `'"{}" ...'.format(font_name)` instead

Reply:
I wonder if we're ready to jump to Python 3.6. Python 2.x finally seems to be nearing the point of deprecation.

Reply:
We're still running this locally, so not yet from our perspective